.travis.yml

```diff
@@ -119,6 +116,9 @@ install:
       cabal new-update head.hackage -v
     fi
   - grep -Ev -- '^\s*--' ${HOME}/.cabal/config | grep -Ev '^\s*$'
+  - (cd /tmp && echo '' | cabal new-repl -w ${HC} --build-dep fail)
+  - if [ $HCNUMVER -ge 80000 ]; then cabal new-install -w ${HC} -j2 --symlink-bindir=$HOME/.local/bin doctest --constraint='doctest ==0.16.*'; fi
+  - if [ $HCNUMVER -eq 80403 ]; then cabal new-install -w ${HC} -j2 --symlink-bindir=$HOME/.local/bin hlint --constraint='hlint ==2.1.*'; fi
```

Comment:
Similarly here.

Reply:
There's an issue about this in `haskell-ci` and I'll look into it (no promises when though).

Reply:
Unfortunately, I'm not willing to accept this PR unless this issue is resolved.

If we do decide to go with this option, I'll add a hack to `travis-maintenance` for this.